al_aws.js

```diff
@@ -216,6 +216,21 @@ var setEnv = function(vars, callback) {
     });
 };
 
+var uploadS3Object = function ({ data, key, bucketName }, callback) {
+    var s3 = new AWS.S3();
+    // Setting up S3 upload parameters
+    const parseData = typeof data !== 'string' ? JSON.stringify(data) : data;
+    const bucket = bucketName ? bucketName : process.env.aws_lambda_s3_bucket
```

Comment:
let's return an error if `bucketName` is not set.
`aws_lambda_s3_bucket ` is for Lambda packages only